src/vs/workbench/common/actions.ts

```diff
@@ -113,7 +113,7 @@ Registry.add(Extensions.WorkbenchActions, new class implements IWorkbenchActionR
 		// otherwise run and dispose
 		try {
 			const from = args?.from || 'keybinding';
-			await actionInstance.run(undefined, { from });
+			await actionInstance.run(args, { from });
```

Comment:
Passing `args` here is safe for that an `undefined` value is never used in all possible action instances I checked.

Reply:
Maybe a reason for this, you might need to change the actions to commands instead. @jrieken?

Reply:
Switching to command is a good choice as long as UI issue mentioned in https://github.com/microsoft/vscode/pull/87813#discussion_r362003344 is resolved. What do you think would be a good solution to it?